requirements-dev.txt

```diff
@@ -3,10 +3,12 @@ coverage
 editorconfig-checker
 flake8
 jupyter
+mypy
 pylint
 sphinx
 sphinx-autobuild
 sphinx-click
 sphinxcontrib-fulltoc
 sphinxcontrib-napoleon
+types-requests
```

Comment:
Do we use `requests` in this repo?

```
$ template git:(mypy) grep -R requests .
./requirements-dev.txt:types-requests
./CONTRIBUTING.md:Any proposed changes to an existing stactools-packages should be done as pull requests. Please make these
./CONTRIBUTING.md:requests against the `main` branch (unless otherwise directed by the dataset maintainer).
```

Reply:
Hmm, yeah `mypy` was complaining about missing `types-requests`. I think I may have had #32 checked out at the time.
I'll remove this and re-evaluate when #32 gets merged.